projects/fusion-e2e/entities/components/inputs/base-input.ts

```diff
@@ -0,0 +1,49 @@
+import {InputTestIdModifiers} from '@ironsource/fusion-ui/entities/test-ids-modifiers';
+import {Locator, Page} from '@playwright/test';
+import {Editable} from '../../behavior';
+import {getTestId} from '../../global/utils';
+
+export class BaseInputComponent extends Editable {
+    private readonly inputTestIdSelector: string;
+    private inputLocator: Locator;
+
+    constructor(page: Page, selector: string) {
+        super(page, selector);
+        this.inputTestIdSelector = getTestId(selector, InputTestIdModifiers.FIELD);
+        this.initializeElement().then(r => r);
+    }
+
+    private async initializeElement() {
```

Comment:
Not sure why you need this method, please create the instance in the constructor